### PR TITLE
12758 Update nginx.conf

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#12758

*Description of changes:*

- updated app_header to allow access to Business Lookup API

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Review by vysakh-menon-aot — APPROVED** [review]

File: nginx.conf [hunk 1]
```diff
@@ -37,7 +37,7 @@ http {
    add_header Pragma "no-cache";
 
    # Content Security Policy
-   add_header Content-Security-Policy "default-src 'self'; frame-src 'self' *.gov.bc.ca *.hotjar.com; script-src 'self' 'unsafe-eval' *.gov.bc.ca *.hotjar.com; style-src 'self' 'unsafe-inline' *.cloudflare.com; font-src 'self' *.gov.bc.ca *.hotjar.com *.cloudflare.com; img-src 'self' *.postescanada-canadapost.ca data:; connect-src 'self' *.gov.bc.ca *.launchdarkly.com *.hotjar.com *.postescanada-canadapost.ca *.sentry.io; manifest-src 'self';";
+   add_header Content-Security-Policy "default-src 'self'; frame-src 'self' *.gov.bc.ca *.hotjar.com; script-src 'self' 'unsafe-eval' *.gov.bc.ca *.hotjar.com; style-src 'self' 'unsafe-inline' *.cloudflare.com; font-src 'self' *.gov.bc.ca *.hotjar.com *.cloudflare.com; img-src 'self' *.postescanada-canadapost.ca data:; connect-src 'self' *.gov.bc.ca *.launchdarkly.com *.hotjar.com *.postescanada-canadapost.ca *.sentry.io *.apigee.net; manifest-src 'self';";
```
Comment (pwei1018) [inline nginx.conf:40]: all 'src' sections need to add the domain.

**Review by rarmitag — APPROVED** [review]

File: nginx.conf (hunk 1 above)
Comment (severinbeauvais) [reply nginx.conf:40]: But this is now the same add_header as Create UI.

Does Create UI need to be updated too?

Comment (severinbeauvais) [reply nginx.conf:40]: Or maybe adding the extra src sections is out of scope for THIS problem?

Comment (pwei1018) [reply nginx.conf:40]: It seems you added the domain into image-src section of the config.

Comment (pwei1018) [reply nginx.conf:40]: Never mind. It is my issue. My eyes get blur.

**Review by pwei1018 — APPROVED** [review]

**Pull request merged**